modules/ROOT/pages/dataweave-formats-flatfile.adoc

```diff
@@ -29,8 +29,116 @@ payload
 
 // CONFIG PROPS GET APPENDED BELOW /////////////////////////////////
 
-// Flat File config properties are shared with COBOL Copybook and Fixed Width.
-include::partial$dataweave-formats-flatfile-config.adoc[]
+
+[[properties]]
+== Configuration Properties
+
+DataWeave supports the following configuration properties for this format.
+
+
+[[reader_properties]]
+=== Reader Properties
+
+This format accepts properties that provide instructions for reading input data.
+
+
```

Comment:
Note: To see if there have been updates since 2.4.0, checked the diff on the support/2.4.0 and support/2.5.0 build of the format docs. There was only one difference, which suggests there was no significant modification.  

```
< |`bufferSize`  |`Number`|`8192`|Size of the buffer writer. The value must be greater than 8.
< 
---
> |`bufferSize`  |`Number`|`8
```192`|Size of the buffer writer. The value needs to be bigger than 8.

Reply:
added to include